includes/class-newspack-popups-model.php

```diff
@@ -294,9 +279,10 @@ public static function retrieve_overlay_test_popups() {
 	/**
 	 * Retrieve first overlay popup matching post categries.
 	 *
+	 * @param  boolean $include_unpublished Whether to include unpublished posts.
 	 * @return object|null Popup object.
 	 */
-	public static function retrieve_category_overlay_popup() {
+	public static function retrieve_category_overlay_popup( $include_unpublished = false ) {
```

Comment:
This method is no longer used with the argument

Reply:
Removed in a603104.